shlr/meson.build

```diff
@@ -286,9 +291,10 @@ qnx_files = [
 qnx_inc = [platform_inc, include_directories('qnx/include')]
 
 librzqnx = static_library('rzqnx', qnx_files,
-  dependencies: [rz_socket_dep, rz_util_dep],
+  dependencies: [rz_util_dep, rz_socket_dep],
   include_directories: qnx_inc,
-  implicit_include_directories: false
+  implicit_include_directories: false,
+  pic: true,
```

Comment:
Actually I need to check whether this is really necessary... maybe by improving the TCC support on meson I did this can be avoided. Need to check better though, but I wanted to start pushing the PR to see if something else broke.

Reply:
Ok indeed this is not actually necessary. I was able to fix the tcc code in meson to avoid this, the same it is done when using other compilers.